code/modules/chemistry/Chemistry-Recipes.dm

```diff
@@ -1923,6 +1923,20 @@ datum
 			result_amount = 3
 			mix_phrase = "An iridescent black chemical forms in the container."
 
+		hemodissolve // denaturing hemolymph
+			name = "Copper"
+			id = "copper"
+			result = "copper"
+			required_reagents = list("hemolymph" = 3, "cleaner" = 1,  "acetone" = 2)// 6 u total to make
+			required_temperature = T0C + 30 // just a little bit of heat
+			result_amount = 1
+			mix_phrase = "The hemolymph denatures into some basal components."
+			on_reaction(var/datum/reagents/holder, created_volume)
+				holder.add_reagent("meat_slurry", 2*created_volume)// 2 meat slurry, since animal tissue
+				holder.add_reagent("saline", created_volume)// 1 saline-glucose solution, since blood
+				holder.add_reagent("spaceacillin", created_volume)// 1 spaceacillin, since hemolymph is used for bacterial tests IRL
+				holder.add_reagent("enzymatic_leftovers", created_volume)// and just some random biological chemicals for good measure
```

Comment:
why not make it match the denaturing heat of booster enzyme, and produce denatured enzymes instead of enzymatic leftovers? Seems like it would feel a bit more consistent

Reply:
That seems like a better idea, considering how denatured enzyme actually has some weird uses